src/Microsoft.ML.Data/Transforms/KeyToVector.cs

```diff
@@ -743,20 +723,56 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, ColInfo info, string src
         }
     }
 
+    /// <summary>
+    /// Estimator for <see cref="KeyToVectorMappingTransformer"/>. Converts the key types back to their original vectors.
+    /// </summary>
     public sealed class KeyToVectorMappingEstimator : TrivialEstimator<KeyToVectorMappingTransformer>
     {
         internal static class Defaults
         {
             public const bool Bag = false;
         }
 
-        public KeyToVectorMappingEstimator(IHostEnvironment env, params KeyToVectorMappingTransformer.ColumnInfo[] columns)
+        /// <summary>
+        /// Describes how the transformer handles one column pair.
+        /// </summary>
+        public sealed class ColumnInfo
+        {
+            /// <summary>
+            /// Name of the column resulting from the transformation of <cref see="InputColumnName"/>.
+            /// </summary>
+            public readonly string Name;
+            /// <summary>
+            /// Name of column to transform. If set to <see langword="null"/>, the value of the <cref see="Name"/> will be used as source.
```

Comment:
>If set to <see langword="null"/>, the value of the <cref see="Name"/> will be used as source. [](start = 45, length = 93)

this does not apply here.

Reply:
Very good point! Thank you :)

---
In reply to: [253750601](https://github.com/dotnet/machinelearning/pull/2364#discussion_r253750601) [](ancestors = 253750601)